index.html

```diff
@@ -64,8 +74,133 @@ <h1>I’m Lilian<br>
+    <div id="item-7"><button class="buttonproject">See Project</button></div>
+  </section>
+
+  <section class="section-grid">
+    <div id="itemb-0"><img src="/images/SnapshootPortfolio2.png" alt="sreenshot portfolio" class="responsive-image">
+    </div>
+    <div id="itemb-1">
+      <p>Multi-Post
+        Stories</p>
+    </div>
+    <div id="itemb-2">
+      <div class="flex-container">
+        <div class="flex-items">
+          <p>CANOPY &nbsp;<img src="/images/Counter.png" alt="canopy"></p>
+        </div>
+        <div class="flex-items">
+          <p>Back End Dev &nbsp;<img src="/images/Counter.png" alt="backend"></p>
+        </div>
+        <div class="flex-items">
+          <p>2015</p>&nbsp;
+        </div>
+        <div class="flex-items"></div>
+      </div>
+    </div>
+    <div id="itemb-3">
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+    </div>
+    <div id="itemb-4"><button class="button button1">html</button>
+    </div>
+    <div id="itemb-5"><button class="button button2">css</button></div>
+    <div id="itemb-6"><button class="button button3">javascript</button></div>
+    <div id="itemb-7"><button class="buttonproject">See Project</button></div>
+  </section>
+
+
+  <section class="about-flex-container">
+    <div class="namee">
+      <h1>About
+        Myself</h1>
+    </div>
+    <div class="descriptionn">
+      <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+        work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+    </div>
+    <div class="connectt">
+      <a href="#" class="connect">LET’S CONNECT</a>
+    </div>
+
+    <div class="social-icons">
+      <ul class="social-icons">
+        <li class="social-icon">
+          <a href="https://twitter.com/ifnotlily"> <img src="images/twitter.svg" alt="Lilian's Twitter" /> </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://www.linkedin.com/in/lilian-francis/">
+            <img src="images/Linkedin.svg" alt="Lilian's LinkedIn" />
+          </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://medium.com/@chatwitlily/">
+            <img src="images/medium.svg" alt="Lilian's Medium" /> </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://angel.co/u/lilian-francis"> <img src="images/angellist.svg" alt="Lilian's AngelList" />
+          </a>
+        </li>
+        <li class="social-icon">
+          <a href="https://github.com/ifnotlily/">
+            <img src="images/github.svg" alt="Lilian's Github" />
+          </a>
+        </li>
+      </ul>
+    </div>
+    <div id="iteme-7"><button class="buttonresume">Get my Resume</button></div>
+    <div class="my-accordion">
+      <ul>
+        <li class='menu'>Langguages<span class='arrow rarrow'></span><span class='arrow darrow'>-</span>
+
```

Comment:
- [ ] There is a small typo is not `Langguages` but rather it should be `Languages`

Reply:
Well resolved